CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 * Fix a typo in the word Year(s) in the Period interval. Refs UICIRC-662.
 * Update `codemirror` and `react-codemirror2`. Refs UICIRC-576.
+. Update `folio/stripes-template-editor` to `3.0.0`. Refs UICIRC-575
```

Comment:
`<really-really-pedantic>`
* `*` not `.` 
* `@folio/stripes-template-editor` not `folio/stripes-template-editor`
`</really-really-pedantic>`